src/style.css

```diff
@@ -2,0 +1,71 @@
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.additional-search-bar {
+  margin-top: 20px;
+}
+
+.search-bar-input {
+  width: 100%;
+  height: 100%;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar-input--main {
+  font-family: Avenir-Book, sans-serif;
+  height: 70px;
+  box-sizing: border-box;
+  background-size: 19px 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 21.86px;
+}
+
+.search-bar-input--additional {
+  font-family: Avenir-Book, sans-serif;
+  height: 42px;
+  box-sizing: border-box;
+  background-size: 11px 11px;
+  background-position: 13px center;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19.12px;
+}
+
+.search-bar-input::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar-input:hover {
+  box-shadow: rgba(0, 0, 0, 0.25);
+}
+
+.search-bar-input:focus {
+  box-shadow: rgba(0, 0, 0, 0.25);
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
The 'font-family' property in the '.search-bar-input:focus' class should include fallback fonts. (Checklist: Remember to use fallback fonts - alternative font-family in case the main one doesn't work)